creator-node/src/services/ContentNodeInfoManager.ts

```diff
@@ -315,9 +315,13 @@ export async function getReplicaSetEndpointsByWallet({
   wallet,
   parentLogger
 }: GetReplicaSetEndpointsByWalletParams): Promise<ReplicaSetEndpoints> {
-  const user: { user_id: number } = (
-    await libs.User.getUsers(1, 0, null, wallet)
-  )[0]
+  const userAsc: { user_id: number }[] = await libs.User.getUsers(
+    1,
+    0,
+    null,
+    wallet
```

Comment:
is there an implicit limit here that will wreck us?

Reply:
good catch yeah the array will always be size 1 if I leave that first limit arg set to 1 🤦 I changed it to 1000, which I think is more than enough. most users only even have 1 result for that route - including you https://discoveryprovider.audius.co/users?handle=rayjacobson

Reply:
i'm the pathological one